src/framework/mlt_log.c

```diff
@@ -82,3 +101,17 @@ void mlt_log_set_callback( void (*new_callback)( void*, int, const char*, va_lis
 {
 	callback = new_callback;
 }
+
+int64_t mlt_log_timings_now( void )
+{
+	int64_t r;
+	struct timeval tv;
+
+	gettimeofday(&tv, NULL);
+	r = tv.tv_sec;
+
+	r *= 1000000LL;
+	r += tv.tv_usec;
+
+	return r;
+}
```

Comment:
How do you feel about making mlt_log_timings_now() a noop if NDEBUG is defined?
http://en.cppreference.com/w/c/error/assert